docs/zap.org

```diff
@@ -1,4 +1,30 @@
-** Zap
-*** Description
-TODO
+*** Zap
+**** Description
+Zap is a micro web framework that is mainly uses to write REST APIs.
+
+Zap isn't really a fully Zig project, it just wraps and leverages the power of [[https://facil.io][facil.io]]. By that I want to emphasize 2 things:
+- All the heavy lifting is done in C, so we can't give much of the performances merits to Zig on this
+- From the programmer POV it seems to you like your project is 100% Zig, so you have no C overhead (except linking libc)
+
+**** Exemples
+The [[https://github.com/zigzap/zap][repo]] gives you a lot of [[https://github.com/zigzap/zap/tree/master/examples][exemples]] in order to get started.
+TODO jaimerais bien faire les exemples ici mais comment faire pour utiliser une librairie dans org babel ? je ne pense pas que se soit possible, du moins facilement
```

Comment:
Check the tangling on orgmode

Reply:
oui jai commence a regarder je verrai demain avec le prof